client/cody/src/status-bar.ts

```diff
@@ -0,0 +1,99 @@
+import * as vscode from 'vscode'
+
+import type { Configuration } from '@sourcegraph/cody-shared/src/configuration'
+
+import { getConfiguration } from './configuration'
+
+export interface CodyStatusBar {
+    dispose(): void
+    startLoading(label: string): () => void
+}
+
+export function createStatusBar(): CodyStatusBar {
+    const statusBarItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right)
+    statusBarItem.text = '$(cody-logo)'
+    statusBarItem.command = 'cody.status-bar.interacted'
+    statusBarItem.show()
+
+    const command = vscode.commands.registerCommand(statusBarItem.command, async () => {
+        const workspaceConfig = vscode.workspace.getConfiguration()
+        const config = getConfiguration(workspaceConfig)
+
+        function createFeatureToggle(
+            name: string,
+            setting: string,
+            getValue: (config: Configuration) => boolean
+        ): vscode.QuickPickItem & { onSelect: () => Promise<void> } {
+            const isEnabled = getValue(config)
+            return {
+                label: (isEnabled ? 'Disable' : 'Enable') + ' ' + name,
+                onSelect: async () => {
+                    await workspaceConfig.update(setting, !isEnabled, vscode.ConfigurationTarget.Global)
+                    await vscode.window.showInformationMessage(name + ' ' + (isEnabled ? 'disabled' : 'enabled') + '.')
```

Comment:
```suggestion
                    const response = await vscode.window.showInformationMessage(name + ' ' + (isEnabled ? 'disabled' : 'enabled') + '.', 'Reload Window')
                    
                    if (response === 'Reload Window') {
                      // Execute reload vscode command
                    }
```

Optional: If it's not trivial to make these work without reloading, we could add a button to the message that prompts users to reload VSCode. Didn't check this code